### PR TITLE
whats-new-12-07-service-level-management-beta

### PR DESCRIPTION
**Pull request by hollorandan**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by barbnewrelic** [comment]
@hollorandan  Does your What's New post still need to be published? It appears to be a draft or a test, is that right? Also, please click the link in your PR to sign the Contributor License Agreement. That is required of all contributors, including New Relic. Thanks!

**Comment by barbnewrelic** [comment]
@hollorandan  We still have not heard back from you about this PR, so we assume it was simply a test. I'll go ahead and close it. You can always reopen it and sign the CLA. Thanks again for being a first time contributor to the docs site!

**Comment by hollorandan** [comment]
Hi there!

Please reopen this - I have no signed the CLA.

Thank you!
Dan

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/whats-new/2021/12/whats-new-12-07-service-level-management-beta.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: 'Get started with New Relic's service level management'
```
Comment (zstix) [inline src/content/whats-new/2021/12/whats-new-12-07-service-level-management-beta.md:2]:
```suggestion
title: Get started with New Relic's service level management
```

**Review by alexiskjones — APPROVED** [review]

**Comment by alexiskjones** [comment]
Hi - I've reviewed this post and it is approved for publishing.

**Pull request merged**